commons-rng-core/src/main/java/org/apache/commons/rng/core/source32/PcgXshRs32.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.commons.rng.core.source32;
+
+/**
+ * XSH RS -- high xorshift, followed by a random shift.
+ * State size is 64 bits and the period is 2<sup>64</sup>.
+ * @see <a href="http://www.pcg-random.org/">
```

Comment:
Note: The state size is 128-bits. Also put a space between the comment and the `@see` tag.

Reply:
The same applies to the other generators too. All have current LCG state and the increment. This is 128-bits of state.

Reply:
Solved.